src/rocm_docs/__init__.py

```diff
@@ -208,15 +208,15 @@ def setup(self) -> None:
                 " readable."
             )
 
-        toc_in_path = self._docs_folder / "./docs/sphinx/_toc.yml.in"
+        toc_in_path = self._docs_folder / "./sphinx/_toc.yml.in"
```

Comment:
change sphinx folder to .sphinx please

Reply:
Done with commit [b6cfe02](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/9/commits/b6cfe0201a4506e87b928dd4eca1094030735ad3)

Also tested with rtd branch for hipBLAS
https://github.com/ROCmSoftwarePlatform/hipBLAS/tree/rtd